--- /usr/share/livecd-rootfs/live-build/auto/config.orig	2025-12-02 22:08:25.876395188 +0000
+++ /usr/share/livecd-rootfs/live-build/auto/config	2025-12-02 22:12:04.999799671 +0000
@@ -786,7 +786,7 @@
 						OPTS="${OPTS:+$OPTS }--initramfs=none"
 						OPTS="${OPTS:+$OPTS }--system=normal"
 						OPTS="${OPTS:+$OPTS }--hdd-label=cloudimg-rootfs"
-						OPTS="${OPTS:+$OPTS }--ext-resize-blocks=536870912 --ext-block-size=4096"
+						OPTS="${OPTS:+$OPTS }--ext-resize-blocks=$((536870912 * 2)) --ext-block-size=4096"
 						OPTS="${OPTS:+$OPTS }--ext-fudge-factor=15"
 						;;
 					intel-iot)
@@ -794,7 +794,7 @@
 						OPTS="${OPTS:+$OPTS }--initramfs=none"
 						OPTS="${OPTS:+$OPTS }--system=normal"
 						OPTS="${OPTS:+$OPTS }--hdd-label=cloudimg-rootfs"
-						OPTS="${OPTS:+$OPTS }--ext-resize-blocks=536870912 --ext-block-size=4096"
+						OPTS="${OPTS:+$OPTS }--ext-resize-blocks=$((536870912 * 2)) --ext-block-size=4096"
 						OPTS="${OPTS:+$OPTS }--ext-fudge-factor=15"
 						;;
 					*)
@@ -1177,7 +1177,7 @@
 		esac
 		OPTS="${OPTS:+$OPTS }--system=normal"
 		OPTS="${OPTS:+$OPTS }--hdd-label=cloudimg-rootfs"
-		OPTS="${OPTS:+$OPTS }--ext-resize-blocks=536870912 --ext-block-size=4096"
+		OPTS="${OPTS:+$OPTS }--ext-resize-blocks=$((536870912 * 2)) --ext-block-size=4096"
 		OPTS="${OPTS:+$OPTS }--ext-fudge-factor=15"
 		;;
 
@@ -1509,9 +1509,11 @@
 			esac
 		fi
 
-		if [ "$IMAGEFORMAT" = none ]; then
-			rm -f config/hooks/*.binary*
-		fi
+                if [ "$IMAGEFORMAT" = none ]; then
+                        if [ "$IMAGE_FORCE_HOOKS" != "true" ]; then
+                                rm -f config/hooks/*.binary*
+                        fi
+                fi
 		;;
 esac
 